frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -356,6 +356,12 @@ function ReaderHighlight:addToMainMenu(menu_items)
         }
     end
     menu_items.translation_settings = Translator:genSettingsMenu()
+
+    if not Device:isTouchDevice() then
+        -- Menu items below aren't needed
```

Comment:
Feels a bit weird to me with an uppercase first letter but no period. :-)

Reply:
OK, added some periods :)
But there must be tons of my small comments without any period. I'm usually adding a period only when I write longer prose with multiple sentences :)